client/src/partials/depots_management/depots_management.js

```diff
@@ -30,6 +30,22 @@ function DepotManagementController($translate, DepotService, SessionService) {
     }
   ];
 
+  /** variables */
+  var map = {
```

Comment:
This is a much cleaner model for doing state management! The map at the top makes it very clear. 

As of https://github.com/IMA-WorldHealth/bhima-2.X/pull/225 we now have [ui-router](https://github.com/angular-ui/ui-router) in the master branch - this should be leveraged (example in the billing services) module for state going forward.